ui/packages/catalog-ui-search/src/main/webapp/component/results/results.less

```diff
@@ -3,14 +3,17 @@
   overflow: hidden;
 
   > .results-select > @{customElementNamespace}dropdown {
+    box-shadow: 2px 2px 3px 0px rgba(0, 0, 0, 0.24);
```

Comment:
Might be worth using `.dropshadow()` or adding a new mixin for this.

Reply:
Good idea. I updated the value while I was in there.